streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -61,14 +61,18 @@
         this.standbyTaskCreator = standbyTaskCreator;
         this.active = active;
         this.standby = standby;
+
+        final LogContext logContext = new LogContext(logPrefix);
```

Comment:
Should be passed in at construction from `StreamThread`

Reply:
`TaskManager` has some exception messages which use `logPrefix` so I passed `logPrefix` instead `LogContext` as `LogContext` can't return `logPrefix`. Please advise.

Reply:
ok - let's leave it as is